python/kserve/requirements.txt

```diff
@@ -0,0 +1,6 @@
+certifi >= 14.05.14
+future; python_version<="2.7"
+six >= 1.10
+python_dateutil >= 2.5.3
+setuptools >= 21.0.0
+urllib3 >= 1.15.1
```

Comment:
Not sure if this file and `python/kserve/setup.py` should be included as they weren't in the repo before.